src/azure-cli/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -317,6 +317,10 @@ def _flexible_server_params(command_group):
             if command_group == 'postgres':
                 c.argument('source_server', options_list=['--source-server'],
                            help='The name of the source server to restore from.')
+                c.argument('source_subscription_id', options_list=['--source-subscription-id'],
```

Comment:
`--source-subscription-id` is too long. Can you consider `--source-sub-id`?

Reply:
We decided to hide this parameters from customers and use same sub id and rg as the source server,, thanks!